site/content/docs/main/upgrade-to-1.11.md

```diff
@@ -33,19 +34,19 @@ Before upgrading, check the [Velero compatibility matrix](https://github.com/vmw
 
     ```bash
     Client:
-        Version: v1.10.0
+        Version: v1.11.0
         Git commit: <git SHA>
     ```
 
-1. Update the Velero custom resource definitions (CRDs) to include schema changes across all CRDs that are at the core of the new features in this release:
+2. Update the Velero custom resource definitions (CRDs) to include schema changes across all CRDs that are at the core of the new features in this release:
 
     ```bash
     velero install --crds-only --dry-run -o yaml | kubectl apply -f -
     ```
 
     **NOTE:** Since velero v1.10.0 only v1 CRD will be supported during installation, therefore, the v1.10.0 will only work on kubernetes version >= v1.16
 
-1. Update the container image and objects fields used by the Velero deployment and, optionally, the restic daemon set:
+3. Update the container image and objects fields used by the Velero deployment and, optionally, the restic daemon set:
 
     ```bash
     # uploader_type value could be restic or kopia
```

Comment:
Should the image version be updated to v1.11.0 below?

Reply:
thanks, I'll update it, I copy it from v1.10 upgrade doc

Reply:
updated